docs/source/cpp/compute.rst

```diff
@@ -1103,6 +1103,8 @@ Containment tests
 +-----------------------+-------+-----------------------------------+----------------+---------------------------------+-------+
 | starts_with           | Unary | Binary- or String-like            | Boolean        | :struct:`MatchSubstringOptions` | \(2)  |
 +-----------------------+-------+-----------------------------------+----------------+---------------------------------+-------+
+| indices_nonzero       | Unary | Boolean, Null, Numeric            | Int64          |                                 |       |
```

Comment:
Please add a description of what this function does, since this is the public reference documentation.

Reply:
Also, add compute function to the [Python compute functions.](https://github.com/apache/arrow/blob/master/docs/source/python/api/compute.rst#containment-tests)

Reply:
Also, it seems like this should be `UInt64`, not `Int64`.

Reply:
👍